packit_service/worker/events/event.py

```diff
@@ -379,29 +381,42 @@ def get_package_config(self) -> Optional[PackageConfig]:
             package_config.upstream_project_url = self.project_url
         return package_config
 
+    @staticmethod
+    def _filter_failed_models_target(
+        models: Union[
+            Optional[Iterable[CoprBuildModel]], Optional[Iterable[TFTTestRunModel]]
+        ],
+    ) -> Optional[Set[str]]:
+        failed_models_target = set()
+        for model in models:
+            if model.status == "failed":
```

Comment:
I recall we have "error" statuses as well, right? if so, should we include them here as well?